### ISSUE TITLE
I'm testing with my dataset, but there are an error that test.py can not read test set.

### ISSUE THREAD

**Issue by seongjunbae** [liuruijin17/LSTR#77]
![iter_0_layer_0_gt](https://user-images.githubusercontent.com/96452534/150287325-59fbcf92-7866-45b3-abb4-2598446e8d7c.jpg)
![iter_0_layer_0_pred](https://user-images.githubusercontent.com/96452534/150287369-650154cd-fa54-4c3f-9d2f-0adc342355f8.jpg)
I think it can train well.(lstr-custom model)
But there are an error in testing.
How can I solve it?

**Comment by wwandji** [comment]
with tool do you use to label? and can i have the script that you use to convert json to txt file? wwankira@gmail.com

**Comment by AvneeshKhanna** [comment]
Hi
Interesting results. Can you tell me what was the size of your training data? And did you change camera intrinsic/extrinsic parameters? if yes, how did you change?
